src/js/lib/monster.ui.js

```diff
@@ -275,6 +275,28 @@ define(function(require) {
 				return monster.ui.getTemplatePhoneNumber(number.toString());
 			},
 
+			renderSvgIcon: function(id, cssClass) {
```

Comment:
We don't need to specify the action as this is a Handlebars helper.
```suggestion
			svgIcon: function(id, cssClass) {
```